react/features/app/components/App.web.js

```diff
@@ -50,6 +50,7 @@ export class App extends AbstractApp {
     componentWillMount(...args) {
         super.componentWillMount(...args);
 
+        this.props.store.dispatch(detectPlatform());
```

Comment:
How may actions will we dispatch in App's componentWillMount. The super dispatches APP_WILL_MOUNT. In a previous PR you added appInit. And now we have detectPlatform. That's more than I'd like to have dispatched right here.